tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -80,15 +80,15 @@ type Parameters = {
  */
 connectionParameters : connection -> Parameters -> Bit
 connectionParameters conn params = params.psk_mode == conn_psk_mode
-                                   /\ params.retry == conn_retry
                                    /\ params.client_auth == (conn_client_auth /\ ~params.psk_mode)
+                                   /\ params.retry == conn_retry
                                    /\ params.no_client_cert == (conn_no_client_cert /\ params.client_auth)
                                    /\ params.zero_rtt == (conn_zero_rtt /\ params.psk_mode)
                                    /\ params.compat_mode == [ conn.mode != S2N_CLIENT, conn.mode == S2N_CLIENT ]
                                    /\ conn.actual_protocol_version == S2N_TLS13
   where conn_psk_mode =         False /* Not supported yet */
-        conn_retry =            False /* Not supported yet */
-        conn_client_auth =      False /* Not supported yet */
+        conn_retry =            conn.handshake.hello_retry_request
+        conn_client_auth =      False /* Shouldn't this be supported now? */
```

Comment:
conn_client_auth hasn't been merged in yet. You should probably keep "Not supported yet" :)

Reply:
Updated.